R/isfiletext.R

```diff
@@ -0,0 +1,34 @@
+#' Determine whether a file is "plain-text" or some sort of binary format
+#'
+#' @param filename Path to the file
+#' @param maxsize  Maximum number of bytes to read
+#' @param textbytes Which characters are used by normal (though not necessarily 
+#'                  just ASCII) text. To detect just ASCII, the following value
+#'                  can be used: `as.raw(c(7:16,18,19,32:127))`
+#' @param tf       If `TRUE` (default) simply return `TRUE` when `filename` 
```

Comment:
I think it would be better to be type safe here and only ever return a logical. What's the use case for returning the non-text bytes?

Reply:
The use-case is debugging. So I got rid of the parameter and instead made it an option. The function will now always return a logical but if the option is set, it will add an attribute listing which bytes disqualified the file from being text.